.ci-builds/.azure-pipelines.yml

```diff
@@ -24,14 +24,27 @@ jobs:
       condition: failed() # only run if the build fails.
       displayName: 'List failures (if any)'
 #-------------------------------------------------------------------------
+- job: ICU4C_Wasm32_Emscripten
+  displayName: 'C: Wasm32 Emscripten'
+  timeoutInMinutes: 60
```

Comment:
It looks like the build for this took 58m 23s to run.
So a timeout value of only 60 mins might be cutting it a bit close...

That said, I'm vaguely concerned that it takes so long to run. This will run on every PR, so it will add a bit of overhead.

FWIW, I didn't add Valgrind runs on every single PR as I thought that a ~38 min build/test was a bit too long... but maybe not. (@srl295 what are your thoughts on it?).

Reply:
Hmm, that is a bit long! 

Could this be part of the 'exhaustive' build and only run occasionaly?